soccer/src/soccer/strategy/agent/position/offense.hpp

```diff
@@ -51,6 +56,32 @@ class Offense : public Position {
 
     // current state of the offensive agent (state machine)
     State current_state_ = IDLING;
+
+    bool scorer = false;
```

Comment:
```suggestion
    bool scorer_ = false;
```
(will require changes everywhere else in .cpp)